### PR TITLE
feat(cli): Manage Team Members

### PR DESCRIPTION
**Pull request by dmurray-lacework**
## Summary

Manage Team Members in the CLI

```
lacework team-members create
lacework team-members list
lacework team-members show <id>
lacework team-members delete <id>
```

## How did you test this change?

run lacework tm create | list | show | delete
add unit test -> go-sdk/cli/cmd/team_members_test.go

test for org level
run lacework tm create | list | show | delete --organization

## Issue

https://lacework.atlassian.net/browse/ALLY-815

### CODE REVIEW

File: cli/cmd/team_members.go [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+			}
+			cli.StopProgress()
+
+			if cli.JSONOutput() {
+				return cli.OutputJSON(response)
+			}
+
+			teamMember := response.Data
+			headers := [][]string{{teamMember.UserGuid, teamMember.UserName, enabled(teamMember.UserEnabled)}}
+			cli.OutputHuman(renderSimpleTable([]string{"GUID", "NAME", "ENABLED"}, headers))
```
Comment (afiune) [inline cli/cmd/team_members.go:111]:
```suggestion
			cli.OutputHuman(renderSimpleTable([]string{"GUID", "NAME", "STATE"}, headers))
```

File: cli/cmd/team_members.go [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+	detailsTable.WriteString("\n")
+	return detailsTable.String()
+}
+
+func promptCreateTeamMember() (string, error) {
+	questions := []*survey.Question{
+		{
+			Name:     "email",
+			Prompt:   &survey.Input{Message: "Email: "},
+			Validate: survey.Required,
```
Comment (afiune) [inline cli/cmd/team_members.go:215]: Can we add a simple validation to this field, at least check that the string has an `@` and a `.`

Comment (afiune) [reply cli/cmd/team_members.go:215]: That way we don't send the request to the APIs to have it return
```
ERROR unable to create team member:
  [POST] https://customerdemo.lacework.net/api/v2/TeamMembers
  [400] validemail: invalid value for userName; username: not a well-formed email address;
```

**Review by afiune — COMMENTED** [review]
Should we add a way to disable team members? Sounds like a good use-case. Thoughts?

**Comment by dmurray-lacework** [comment]
> Should we add a way to disable team members? Sounds like a good use-case. Thoughts?

I think in perhaps another ticket. I don't know what that command looks like atm. I think we can add some ideas to a new ticket.

File: cli/cmd/team_members.go [hunk 3]
```diff
@@ -0,0 +1,403 @@
...
+		Long:  "Delete a single team member by it's ID.",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+			cli.StartProgress(" Deleting team member...")
+			err := cli.LwApi.V2.TeamMembers.Delete(args[0])
+			cli.StopProgress()
+			if err != nil {
+				return errors.Wrap(err, "unable to delete team member")
+			}
+			cli.OutputHuman("The team member with GUID %s was deleted\n", args[0])
```
Comment (afiune) [inline cli/cmd/team_members.go:133]: Could we agree on a pattern for delete operations? We can clearly see that many engineers print this message in different forms:
```
cli/cmd/alert_rules.go:                 cli.OutputHuman(fmt.Sprintf("The alert rule with GUID %s was deleted \n", args[0]))
cli/cmd/integration.go:                 cli.OutputHuman("The integration %s was deleted.\n", args[0])
cli/cmd/lql_delete.go:  cli.OutputHuman("The query %s was deleted.\n", args[0])
cli/cmd/policy.go:              fmt.Sprintf("The policy %s was deleted.\n", args[0]))
cli/cmd/report_rules.go:                        cli.OutputHuman("The report rule with GUID %s was deleted\n", args[0])
cli/cmd/team_members.go:                        cli.OutputHuman("The team member with GUID %s was deleted\n", args[0])
```

**Comment by dmurray-lacework** [comment]
> When creating a user at the organization level I get:
> 
> Screen Shot 2022-02-01 at 8 02 03 AM
> 
> ```
> ERROR unable to create team member: client configured to manage account-level datasets, use Create()
> ```

We require the --organization flag for this. Should we set org header if the user selects the organzation in the prompt?

**Pull request merged**